src/app/constants/regular-expressions.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const RegularExpressions = {
```

Comment:
```suggestion
export const REGULAR_EXPRESS = {
```

For constants, variable name must be in uppercase check everywhere